website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -41,7 +41,7 @@ The following fields are required when creating a Redshift connection:
 
 ### Connecting via an SSH Tunnel
 
-To connect to a Postgres or Redshift instance via an SSH tunnel, check the "Use SSH Tunnel" option when creating your connection. When configuring the tunnel, you'll need to supply the hostname, username, and port for the bastion server.
+To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, check the "Use SSH Tunnel" option when creating your connection. When configuring the tunnel, you'll need to supply the hostname, username, and port for the bastion server.
```

Comment:
```suggestion
To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, select the **Use SSH Tunnel** option when creating your connection. When configuring the tunnel, you must supply the hostname, username, and port for the bastion server.
```

Reply:
the docs team is working on a style guide, which we've started to use (and will publish soon!). for checkboxes, "select" and "choose" are the preferred words to use: https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md#checkboxes